### PR TITLE
Fix-#813 Receipt link Fixed

### LINKED ISSUE #813: No intent for Opening Receipt Link in Receipt
**Summary:**

On clicking the receipt link in the receipt, it does not open the webpage. Only Copy feature works.

**Steps to reproduce:**

1. Select Payments
2. Select History
3. Click on any transaction
4.View Receipt
5. Click on Receipt link
**Expected behavior:**
It should have opened the webpage for receipt
**Observed behavior:**
But it does nothing on clicking.

**Device and Android version:**
Redmi 7A Android 9
**Screenshots:**
ezgif-3-0b7dce33494b

### PR DESCRIPTION
**Pull request by garvit984**
## Screenshots
ezgif-3-55f645e0551a

## Description
I have now applied an intent for a single click on the link.
##

- [ ] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [ ] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [ ] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

**Comment by dubeyaayush07** [comment]
duplicate #786

**Comment by Bhavnaharitsa** [comment]
@garvit984 Please squash the commits into a single commit and also give a little description about the issue.  
https://gist.github.com/patik/b8a9dc5cd356f9f6f980 -- This link will help you :)

**Comment by Grandolf49** [comment]
@garvit984 You have 1 PMD violation in your PR.
Check it here 
**mobile-wallet/mifospay/build/reports/pmd/pmd.html**

File: mifospay/src/main/java/org/mifos/mobilewallet/mifospay/receipt/ui/ReceiptActivity.java [hunk 1]
```diff
@@ -92,6 +93,17 @@ protected void onCreate(Bundle savedInstanceState) {
...
                 transactionId = params.get(0); // "transactionId"
                 tvReceiptLink.setText(data.toString());
+                tvReceiptLink.setOnClickListener(new View.OnClickListener() {
+                    @Override
+                    public void onClick(View v) {
+                        Uri webpage = Uri.parse(tvReceiptLink.getText().toString());
+                        Intent intent = new Intent(Intent.ACTION_VIEW, webpage);
+                        if (intent.resolveActivity(getPackageManager()) != null) {
+                            startActivity(intent);
+                        } else
```
Comment (robustTechie) [inline mifospay/src/main/java/org/mifos/mobilewallet/mifospay/receipt/ui/ReceiptActivity.java]: Curly bracket is missing after else

Comment (robustTechie) [reply mifospay/src/main/java/org/mifos/mobilewallet/mifospay/receipt/ui/ReceiptActivity.java]: This may also generate PMD violations

**Comment by robustTechie** [comment]
@garvit984  is there any extra import statement you imported?

**Comment by garvit984** [comment]
No @robustTechie

**Comment by shiv07tiwari** [comment]
@garvit984 Can you please explain the use case of this issue?

**Comment by shiv07tiwari** [comment]
@garvit984 We do not need to open the deeplink in any browser. It is meant to open a segment directly in the app.

**Pull request closed without merging**